charts/trento-server/Chart.yaml

```diff
@@ -1,13 +1,13 @@
 # SPDX-License-Identifier: Apache-2.0
-#!BuildTag: trento/trento-server:2.4.1
-#!BuildTag: trento/trento-server:2.4.1-build%RELEASE%
+#!BuildTag: trento/trento-server:2.4.2-dev1
```

Comment:
I'm not sure about the dev1 part
why not 2.4.1-dev1? 
i know we bump them up every time, but what do we do in between releases?

Reply:
it's not considered a version bump, I am afraid https://github.com/trento-project/helm-charts/actions/runs/13310013635/job/37169923273

Reply:
I think `2.4.2-dev1` was correct. The minus says that this version is less thatn 2.4.1, and that's not true